moped-api/app.py

```diff
@@ -21,7 +20,6 @@
 #
 app.register_blueprint(auth_blueprint, url_prefix="/auth")
 app.register_blueprint(users_blueprint, url_prefix="/users")
-app.register_blueprint(events_blueprint, url_prefix="/events")
```

Comment:
So glad that we went with blueprints here - hopefully it made tearing this out a little bit easier.